api/v1alpha1/kafkacluster_types.go

```diff
@@ -192,12 +191,12 @@ func (bConfig *BrokerConfig) GetResources() *corev1.ResourceRequirements {
 	}
 	return &corev1.ResourceRequirements{
 		Limits: corev1.ResourceList{
-			"cpu":    resource.MustParse("500m"),
-			"memory": resource.MustParse("1Gi"),
+			"cpu":    resource.MustParse("1500m"),
```

Comment:
I would make these resource requirements configurable through the CR. People usually have different usage patterns.

Reply:
You are absolutely right, but we are planning to deal with this on a separate PR. This one is getting huge

Reply:
sure thing